From 22a8ff8fcbe002a0d76cd21a1aadf52dc998cc22 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sun, 15 Jan 2017 19:15:19 +0100
Subject: [PATCH 5/7] Turn an SGML comment in menu-policy into an actual
 paragraph

This comment describes how the document is supposed to be maintained. So
it seems relevant in the About chapter.

This also makes it easier to convert to DocBook, as otherwise the comment
gets lost, and it cannot be mangled as it is located in a place where a
<para> is not valid.
---
 menu-policy.sgml | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/menu-policy.sgml b/menu-policy.sgml
index c919740..2dbfee5 100644
--- a/menu-policy.sgml
+++ b/menu-policy.sgml
@@ -5,12 +5,6 @@
 ]>
 <debiandoc>
 
-  <!--
-  The debian-policy mailing list has taken responsibility for the
-  contents of this document, with the package maintainers responsible
-  for packaging administrivia only.
-  -->
-
   <book>
     <titlepag>
       <title>The Debian Menu sub-policy</title>
@@ -95,6 +89,13 @@
 	  </item>
 	</enumlist>
       </p>
+
+      <p>
+        The <url id="mailto:debian-policy@lists.debian.org"
+        name="debian-policy mailing list"> has taken responsibility for
+        the contents of this document, with the <em>Menu</em> package
+        maintainer's responsible for packaging administrivia only.
+      </p>
     </chapt>
 
     <chapt>
-- 
2.11.0

